debian/control

```diff
@@ -38,6 +38,7 @@ Build-Depends: bc,
                libldap2-dev,
                libleveldb-dev,
                liblttng-ust-dev,
+               liblz4-dev,
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, could you prefix it with "debian: " in this case?